### ISSUE TITLE
Make Serbo-Croatian's flag Yugoslavia's flag

### ISSUE THREAD

**Issue by Kxeo** [hugolabe/Wike#187]
I have no idea why it is BiH's flag. Why not Yugoslavia's flag? While yes, Slovenia and Macedonia don't speak (any dialect of) Serbo-Croatian at all, and Kosovo _mostly_ doesn't speak Serbo-Croatian, still, it seems that most of the former Yugoslavia area spoke Serbo-Croatian. Croatia, Bosnia, Herzegovina, Serbia and Montenegro all spoke Serbo-Croatian. Maybe it's BiH's flag because it spoke the language? idk.

**Comment by hugolabe** [comment]
Sorry for the delay in replying @Kxeo 

In some cases it is difficult to choose which flag to assign to a language, since there is no direct correspondence between languages ​​and countries.

Do you think it would be more appropriate to assign the flag of Yugoslavia to the Serbo-Croatian language? For my part, I have no problem changing it, but I don't have a great knowledge of the languages ​​spoken in that area.

**Comment by Kxeo** [comment]
> Do you think it would be more appropriate to assign the flag of Yugoslavia to the Serbo-Croatian language?
Yes, I do. On top of the point that I mentions in the initial post, Serbo-Croatian was the main language of Yugoslavia when it was whole as well. At the time it was considered a singular language. Some say it still is.
In my mind it just makes sense to use the Yugoslav flag for Serbo-Croatian. It's not perfect, but at the same time, just having Bosnia-Herzegovina's flag for Serbo-Croatian is slightly worse. Despite it having A Serbo-Croatian-speaking country and not anybody else.

**Comment by hugolabe** [comment]
Done 089a9cd54bb6b66ee556d49ea13145caf0e9e6d2
